deploy-board/deploy_board/webapp/helpers/placements_helper.py

```diff
@@ -36,3 +36,13 @@ def get_by_provider_and_cell_name(request, provider, cell_name):
 
 def get_by_id(request, placement_id):
     return rodimus_client.get("/placements/%s" % placement_id, request.teletraan_user_id.token)
+
+
+def get_simplified_by_ids(request, placement_str, provider, cell):
+    current_placement_ids = set(placement_str.split(','))
+    all_placements = get_by_provider_and_cell_name(request, provider, cell)
+    current_placements = filter(lambda p: p['id'] in current_placement_ids, all_placements)
+    return map(lambda p: {'id': p['id'],
+                          'capacity': p['capacity'],
+                          'abstract_name': p['abstract_name'],
```

Comment:
might prefer `p.get(k, '')` since this is just for the user. then missing keys will be populated ''